### PR TITLE
Starter project

### PR DESCRIPTION
**Pull request by Chronos34**
This PR contains the jetty 9 server fixes. I have implemented the asynchronous function required for the week. It displays one of three random messages as the last statement to my profile.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -33,3 +33,13 @@ function addRandomGreeting() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+async function welcomeMessage() {
```
Comment (tatkins1) [inline portfolio/src/main/webapp/script.js:37]: Very nice usage of async/await

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -33,3 +33,13 @@ function addRandomGreeting() {
...
+
+    const message = await fetch('/hello');
+    const response = await message.json(); 
+    const randomMessage = response[Math.floor(Math.random() * response.length)];
```
Comment (tatkins1) [inline portfolio/src/main/webapp/script.js]:
```suggestion
    const response = await fetch('/hello');
    const messages = await response.json(); 
    const randomMessage = messages[Math.floor(Math.random() * response.length)];
```

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 3]
```diff
@@ -10,9 +13,14 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+    private final Gson gson = new Gson();
+    private final ArrayList<String> message = new ArrayList<String>(
+        Arrays.asList("Welcome To Fortress Inquisitorius!",
+        "It's a beautiful day!", "Have a Starsome Day!"));
```
Comment (tatkins1) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: When working with long lists add each item on a new line for improved readability.
```suggestion
        Arrays.asList("Welcome To Fortress Inquisitorius!",
        "It's a beautiful day!",
        "Have a Starsome Day!"
        ));
```

**Review by tatkins1 — APPROVED** [review]
Good job!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (Chronos34) [reply portfolio/src/main/webapp/script.js:37]: Thank you!

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 3 above)
Comment (Chronos34) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Noted!

**Pull request merged**